Specs/DateTime/Dutch/TimePeriodParser.json

```diff
@@ -1156,20 +1123,19 @@
           }
         },
         "Start": 7,
-        "Length": 20
+        "Length": 18
       }
     ]
   },
   {
-    "Input": "Ik heb van 0:01 tot 1 gewacht",
+    "Input": "Ik heb gewacht van 0:01 's ochtends tot 1",
```

Comment:
@iMicknl ?

Reply:
Position of `gewacht` doesn't matter, however I don't think that we should always include the `morning` suffix. I didn't go through all test cases, but in my opinion both of them should be supported.

Not sure how `1 tot 4` should be parsed, it makes sense to use `1:00 + 13:00 `and `4:00 + 16:00`

Reply:
added spec "Ik heb gewacht van 0:01 tot 1", it is parsed as (T00:01,T01,PT59M)